config/config.proto

```diff
@@ -90,6 +90,16 @@ message FlowdockConfig {
   optional bool send_resolved = 4 [default = false];
 }
 
+// Configuration for notification via OpsGenie.
+message OpsGenieConfig {
+  // https://api.opsgenie.com/v1/json/alert
+  optional string webhook_url = 1;
+  // See (https://www.opsgenie.com/docs/web-api/alert-api).
+  optional string api_key = 2;
+  repeated string teams = 3;
+  optional bool send_resolved = 4 [default = false];
```

Comment:
(Ok, I guess the link covers them all, but the doc comments here will actually be transformed into Go doc comments on each field in the generated protobuf structs, and then they will seem missing).